src/main/java/mate/academy/internetshop/controllers/order/DeleteOrderController.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.internetshop.controllers.order;
```

Comment:
```suggestion
package mate.academy.internetshop.controller.order;
```
назви пекеджів пишуть в однині

Reply:
I can’t understand exactly which point is not fulfilled

Reply:
package renamed)